tests/integration/fuzzing/parameterized_test.cairo

```diff
@@ -0,0 +1,25 @@
+%lang starknet
+
+from starkware.cairo.common.math import assert_le
+
+@external
+func setup_examples() {
+    %{
+        max_examples(5)
+        # todo test no example
+        example(1, 2)
+        example(2, 3)
+        example(a=3, b=4)
+        example(b=6, a=5)
+        given(a = strategy.integers(15, 20), b = strategy.integers(10, 14))
+    %}
+    return ();
+}
+
+@external
+func test_examples{syscall_ptr: felt*, range_check_ptr}(a: felt, b: felt) {
+    assert_le(0, a);
+    assert_le(0, b);
```

Comment:
[Tests] Test checking if the specified example is actually provided to the test case is missing.

Reply:
[check number of test runs](https://github.com/software-mansion/protostar/pull/947/commits/7b224466a0986a856f952a105af472d4b2ca7737)

Reply:
@karol-bisztyga you can if you manipulate the `phases` setting https://hypothesis.readthedocs.io/en/latest/settings.html#hypothesis.settings.phases

Reply:
now that you manipulate `phases`, you should be able to write a test as suggested by @kasperski95 :)

Reply:
[add a check for the exact example value](https://github.com/software-mansion/protostar/pull/947/commits/d60dcaa9c152af075bfeb0964477879fcb4da6a6)